<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pg_gestao_vendas</class>
 <widget class="QDialog" name="pg_gestao_vendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_filtrar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pdf">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar PDF</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabela_vendas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>851</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabela_produtos_venda">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>851</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mostrar_vendas">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar todas as Vendas</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>10</y>
     <width>261</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="data_inicial">
        <property name="text">
         <string>Data Inicial</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_inicial"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="data_final">
        <property name="text">
         <string>Data Final</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_final"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="data_atual">
      <property name="text">
       <string>Data Atual</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="data_atual_2">
      <property name="text">
       <string>Data Atual</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
